server/routerlicious/packages/services-client/src/interfaces.ts

```diff
@@ -50,4 +50,11 @@ export interface ISession {
 	 * Activity time window is defined by `DefaultServiceConfiguration.documentLambda.partitionActivityTimeout`.
 	 */
 	isSessionActive: boolean;
+
+	/**
+	 * Whether the session stickiness should be ignored during session discovery.
+	 * Session stickiness is ignored if the landed cluster is in draining process.
+	 * Session should be landed on a new cluster immediately if the session stickiness is ignored.
+	 */
+	ignoreSessionStickiness?: boolean;
```

Comment:
Nit: Presumably undefined === false here. It would be good to note that via a `@defaultValue` comment block.